index.html

```diff
@@ -17,7 +17,7 @@
             <a href="/" class="logo">Onomeh </a>
             <nav class="navbar">
               <button class="nav-button" type="button">
-                <img src="images/Union.svg" alt="more" />
+                <img src="assets/Icons/hamburger-button.svg" alt="more" />
               </button>
             </nav>
           </div>
```

Comment:
- [ ] Kindly fix the place of the hamburger menu. It should be on the right of the page. Keep rocking 💪 
- [ ] Please fix the background of the first section following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) because I can't see it.
Here you have a screenshot of the first section:
![image](https://user-images.githubusercontent.com/74506933/153711157-caffc2c6-6f0b-4b58-973b-2e7ffdbdd67e.png)